source/mir/random/engine/mersenne_twister.d

```diff
@@ -105,7 +106,7 @@ struct MersenneTwisterEngine(UIntType, size_t w, size_t n, size_t m, size_t r,
         // them separately in sequence, the variables
         // are kept 'hot' in CPU registers, allowing
         // for significantly faster performance.
-        sizediff_t index = cast(size_t)this.index;
+        sizediff_t index = this.index;
```

Comment:
struct methods accept the struct by reference. So I want compiler to do one load and one store of index per opCall. Overriding allows to force to use `this.` to get `index` of struct.